### PR TITLE
Extract fields that have the same copy separately

### LINKED ISSUE #253: When two fields have the same copy, content from only one is extracted
Just noticed that issue with the prod homepage of https://give.thunderbird.net, because staging had no `seo_title` set.

The `Page title` (or `seo_title`) field in the Promote tab has the following copy: `Give To Thunderbird`

The `Title` field in the Content tab (so, the actual page title) has the same copy, `Give To Thunderbird`

Only the seo_title got extracted in the .pot file: https://github.com/mozilla-l10n/thunderbird-donate-content/blob/master/templates/pages/mozilla-donate.pot#L8

We should have had something like this:
```
msgctxt "seo_title"
msgid "Give to Thunderbird"
msgstr ""

msgctxt "title"
msgid "Give to Thunderbird"
msgstr ""
```

### PR DESCRIPTION
**Pull request by kaedroho**
Fixes #253

### CODE REVIEW

File: wagtail_localize/models.py [hunk 1]
```diff
@@ -363,10 +363,10 @@ def export_po(self):
         Exports a PO file contining the source strings.
         """
         # Get messages
-        messages = defaultdict(list)
+        messages = set()
```
Comment (KalobTaulien) [inline wagtail_localize/models.py:366]: Keep in mind a `set()` won't maintain its ordering.

**Review by KalobTaulien — APPROVED** [review]

File: wagtail_localize/models.py (hunk 1 above)
Comment (kaedroho) [reply wagtail_localize/models.py:366]: Oh, I assumed they worked the same way as a dict! thanks for spotting, we definitely want that ordered

Comment (kaedroho) [reply wagtail_localize/models.py]: Fixed, I just turned it into a list. There isn't really a need to deduplicate things anyway!

**Comment by TheoChevalier** [comment]
Hey @kaedroho, should this be just merged, or is there more work needed?

**Pull request merged**